src/app/components/LanguageInfo.svelte

```diff
@@ -119,7 +140,17 @@
       cyData="input-copyright"
       subtext="" />
   </div>
-
+  <div class="copyright-field">
+    {#if copyrightButtonEnabled}
+      <Button
+        size="small"
+        color="blue"
+        isOutlined={true}
+        onClick={addCopyrightSymbol}>
+        {$_('common.add') + ' ©'}
```

Comment:
It's better to translate the _entire_ button text.

```suggestion
        {$_('common.add_copyright')'}
```

I will not block the PR since it requires a decent amount of work, but not all languages will have a 1-to-1 translation for the word "add"; and definitely not all languages place the object after the verb as in "Add[verb] ©[object]". Not all languages use spaces either. This is the Google Translate for "Add ©" (unverified, I do not know Japanese, I just listen to a lot of City Pop):

> ©を追加

Notice how © goes first, then there's no space between that and  "を" (-o), the object marker, and 追加 a verb for adding (I'm not sure if this is the right verb!)